todolist/settings.py

```diff
@@ -66,7 +66,7 @@
         'NAME': 'app_db',
         'USER': 'app_user',
         'PASSWORD': '1234',
```

Comment:
The database password is hardcoded in the settings file. It's recommended to use environment variables to manage sensitive information like database credentials.